src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml

```diff
@@ -0,0 +1,1194 @@
+                                                           IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+
+                                <controls:CalculatorButton x:Name="InvsechButton"
+                                                           x:Uid="invsechButton"
+                                                           Grid.Row="1"
+                                                           Style="{StaticResource OperatorButtonStyle}"
+                                                           AutomationProperties.AutomationId="invsechButton"
+                                                           ButtonId="InvSech"
+                                                           Click="FlyoutButton_Clicked"
+                                                           Content="sech⁻¹"
+                                                           IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+
+                                <controls:CalculatorButton x:Name="InvcschButton"
+                                                           x:Uid="invcschButton"
+                                                           Grid.Row="1"
+                                                           Grid.Column="1"
+                                                           Style="{StaticResource OperatorButtonStyle}"
+                                                           AutomationProperties.AutomationId="invcschButton"
+                                                           ButtonId="InvCsch"
+                                                           Click="FlyoutButton_Clicked"
+                                                           Content="csch⁻¹"
+                                                           IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+
+                                <controls:CalculatorButton x:Name="InvcothButton"
+                                                           x:Uid="invcothButton"
+                                                           Grid.Row="1"
+                                                           Grid.Column="2"
+                                                           Style="{StaticResource OperatorButtonStyle}"
+                                                           AutomationProperties.AutomationId="invcothButton"
+                                                           ButtonId="InvCoth"
+                                                           Click="FlyoutButton_Clicked"
+                                                           Content="coth⁻¹"
+                                                           IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+                            </Grid>
+                        </Grid>
+                    </Flyout>
+                </controls:OperatorPanelButton.FlyoutMenu>
+            </controls:OperatorPanelButton>
+
+            <controls:OperatorPanelButton x:Name="InequalityButton"
```

Comment:
><controls:OperatorPanelButton x:Name="InequalityButton" [](start = 12, length = 55)

I don't think the graphing engine supports inequalities right now.  We should probably remove this until it does.

Reply:
It does support inequalities! Try typing `x>6` into an equation